### ISSUE TITLE
Make 'master' the default branch?

### ISSUE THREAD

**Issue by wdlkmpx** [puppylinux-woof-CE/woof-CE#1200]
Releases should use the master branch.. everything created with some other branch would be 'experimental'..

So that interesting changes can pile up in testing until they are properly tested and become more or less 'stable'.

what do you think...

**Comment by wdlkmpx** [comment]
I think things were pretty broken for xenial, major changes had to happen before xenial could be properly supported. 

And in fact things are still broken here and there. But now a release would be ok. The xenial listed in puppylinux.com has some bad bugs.

Unless the same PPM bug i see in dpup is in xenial.. then a new release would  not make sense yet.

I was thinking of automatic builds, not really automatic but someone who runs a script and wastes resources for a couple hours building at least slacko142-32/64 and xenialpup 32/64 and uploading them to some server, so tests can be carried out. I read your users can do that.

I might do that, but not this month.. that was the reason i added (the now deleted) ISOS page.

**Comment by wdlkmpx** [comment]
I want testing to be the development branch where bombs are dropped without much concern. In the past i spent a lot of time testing the changes i made, they were usable when they landed on testing, and so only had to fix a few things.  But that won't happen in the future, but i want to keep pushing my agenda forward, and this will benefit woofce of course. I use no other linux distro, had i known about what the dogs actually were back in early 2015, i would not be here, but i am, and have to continue but i know my changes are all good, so why not continue..

Basically all major changes, no matter how much they are tested by the developer... they will have many bugs...

Lxpupsc is ok for testing, experiments are ok for testing. As you're the only one who has sacrificed many ISOS for the cause, stuff gets tested and fixed in the lxpups first.

But for more serious releases, and in the puppy land, official releases happen once a pale moon, so that has to happen in master with stuff more or less stable.. because puppy is always unstable, at least for me...

**Comment by wdlkmpx** [comment]
The 'testing' branch is ready to be merged to master, or so i think. However that will happen next week.

A new variable will be introduced to DISTRO_SPECS... 'WOOF_VERSION'. Because DISTRO_VERSION means nothing nowadays...
